app/jobs/send_assessments_job.rb

```diff
@@ -8,11 +8,14 @@ def perform(*_args)
     eligible = Patient.reminder_eligible.count
     sent = []
     not_sent = []
-    Patient.reminder_eligible.find_each do |patient|
+    Patient.reminder_eligible.find_each.with_index do |patient, indx|
       sent << { id: patient.id, method: patient.preferred_contact_method } if patient.send_assessment
     rescue StandardError => e
       not_sent << { id: patient.id, method: patient.preferred_contact_method, reason: e.message }
       next
+    ensure
+      # Trigger garbage collect every 2,000 patients
+      GC.start if (indx % 2_000).zero?
```

Comment:
What was the reason for this change in this PR?

Reply:
On the GitLab runner that is doing the job I found that ruby was not garbage collecting fast enough, eats up all the memory, and then gets killed by the system before it can finish. This kicks off garbage collection every 2,000 patients to avoid this issue. 

As more support for this, I'm running the job on my local machine with the 1M database and it does the same without GC being invoked manually (currently @ 25GB and rising, while with GC it hovers somewhere between 0.5 and 1GB).
<img width="964" alt="Screen Shot 2021-02-18 at 8 06 54 AM" src="https://user-images.githubusercontent.com/24628687/108376467-44641000-71c0-11eb-9745-77ad28f9cf7d.png">